build.sh

```diff
@@ -24,7 +24,7 @@ $EM_DIR/emconfigure ./configure \
     --disable-workshop \
     --disable-netbeans \
     --disable-sniff \
-    --disable-multibyte \
+    --enable-multibyte \
```

Comment:
Although I do think that unicode support is important, 44MB size bothers me, at least it can't be a good size for an online service.
So I'd say leave it disabled by default.